source/Angle.cpp

```diff
@@ -164,6 +175,26 @@ Point Angle::Rotate(const Point &point) const
 
 
 
+// Judge whether this is inside from "base" to "limit."
+// The range from "base" to "limit" is expressed by "clock" orientation.
+bool Angle::IsInRange(const Angle& base, const Angle& limit) const
+{
+	// Choose an edge of the arc as the reference angle (base) and
+	// compare relative angles to decide whether this is in the range.
+	Angle normalizedLimit = limit - base;
+	Angle normalizedTarget = *this - base;
+	return normalizedTarget.angle <= normalizedLimit.angle;
+}
+
+
+
+bool Angle::IsInRange(const pair<Angle, Angle>& range) const
+{
+	return IsInRange(range.first, range.second);
+}
```

Comment:
do we need this? cant we just split it when we give it to the other IsInRange?
seems like an unnecessary function

Reply:
also I found no point where you used it...

Reply:
It's from the previous PR, maybe it was used back then but no longer used after whatever mess I made with it. I'll get rid of it later when I have time.